.github/workflows/linux_pr.yml

```diff
@@ -18,12 +18,11 @@ jobs:
         version: 5.14.0
         modules: qtcharts
     - name: Configure and compile MNE-CPP
-      run: |
-        qmake -query    
+      run: |    
         qmake -r MNECPP_CONFIG+=noTests
         make -j2 
 
-  ReleaseQtMinimum:
+  QtMinimum:
```

Comment:
There are entries for QtLatest (above) and QtMinimum here both with run-on: ubuntu-latest   Above there is Linux: then indented, runs-on: Ubuntuu 16-04.   I wish there was a way to define as variables QtLatest and QtMinimum and then subsequently use them as was follows "runs-on".   Or I would assume QtMinimum shouls always be Ubuntu 16-04

Reply:
Are you maybe referring to this feature, see [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix).

Reply:
OK - looks ike the idea is to allow setup of an OS matrix and then different OS can run on a specific node.